lfs/config.go

```diff
@@ -544,6 +565,38 @@ func (c *Configuration) readGitConfig(output string, uniqRemotes map[string]bool
 	}
 }
 
+// Special handling for url.<url>.insteadOf, since <url> will contain '.'
+func (c *Configuration) readGitInsteadOfConfig(output string, uniqInsteadOfs map[string]string) {
+	lines := strings.Split(output, "\n")
+	uniqKeys := make(map[string]string)
+	for _, line := range lines {
+		// Why is this not "="?
+		pieces := strings.SplitN(line, " ", 2)
```

Comment:
No, I meant why the string we're splitting on is not "=": inconsistency in the way `git config` reports; see line 489.